create_aio_app/template/{{cookiecutter.project_name}}/Makefile

```diff
@@ -43,7 +43,14 @@ migrations:
 
 migrate:
 	@docker exec -it {{ cookiecutter.project_name }} alembic -n alembic:dev upgrade head;
+
+wait_postgres:
```

Comment:
we talked about that script will be wait not only postgres, so i think will be better rewrite to `wait_resources`